config/webpack.config.js

```diff
@@ -56,6 +56,7 @@ const globalEntryFiles = {
   vendor: sharedModules,
   // This is to solve the issue of the vendor file being cached
   'shared-modules': sharedModules,
+  'web-components': getAbsolutePath('src/platform/site-wide/wc-loader.js'),
```

Comment:
This allows us to use `web-components` as a `data-entry-name` attribute. It will add the appropriate `src` in the build process.